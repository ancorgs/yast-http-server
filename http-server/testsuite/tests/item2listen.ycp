{
	include "testsuite.ycp";

	boolean Modified() ``{
    		return true;
	};

	boolean AbortFunction() ``{
		return true;
	}


	include "http-server/routines.ycp";

	TEST( ``{ return item2listen( "192.168.1.2" , "80"); }, [ $[], $[], $[] ], nil );
}
