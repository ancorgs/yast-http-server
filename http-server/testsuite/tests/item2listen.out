Read	.target.tmpdir nil
Log	Failed to set temporary directory: nil
Return	192.168.1.2:80
